_data/video.yml

```diff
@@ -19,9 +19,29 @@
 
 - url: https://www.youtube.com/watch?v=XvoHY3QCJtw
   title: Labyrinth
-  img: labyrinth.jpg
+  img: verma-labyrinth.png
   author_name: Yash Kumar Verma
   
+- url: https://www.youtube.com/watch?v=yc_d5JJ--vQ
+  title: SUSI AI on Kubernetes
+  img: verma-susu-kubernetes.png
```

Comment:
Everything seems to be working but I think you can use a better file name here. I mean why `susu`?

Reply:
typo :laugh:

Reply:
Haha, I knew. :laughing:

Reply:
Btw images are well made, :+1: